monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md

```diff
@@ -0,0 +1,129 @@
+**Monero en circulation :**16,570,126 XMR (Approximativement)
+**Capitalisation de marché :**$1,503,928,296 USD (~0.77% de la capitalisation de marché totale des cryptomonnaies)
+**Récompense de bloc actuelle :**3.5984 XMR
+**Interval de bloc moyen :**2 Minutes
+
+_Le tarifde la récompense va constamment diminuer jusqu'à fin mai 2022, quand il y aura 18,132 million de XMR en circulation, à ce moment-là, une récompense de bloc de 0,6 XMR demeurera indéfiniment._
+
+_Avec « l'émission résiduelle » de 0,6 XMR/bloc, d'ici 2040 il y aura un nombreéquivalent de Monero que de Bitcoin (environ 21 million)_
+
+### FONCTIONNALITÉS EN DÉVELOPPEMENT
+
+Bien que Monero soit déjà disponible et utilisé à travers le globe, la communauté de développeurs a de passionnants objectifs pour continuer à améliorer la confidentialité, la sécurité, et l'ergonomie de Monero et des cryptomonnaies en général. En voici quelques-unes qui arriveront prochainement :
+
+**Kovri :** C'est une amélioration majeure de la confidentialité, qui ajoute un quatrième niveau de confidentialité aux transactions Monero. Kovri utilise du **[chiffrement en ail](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (pensez à Tor sous stéroïdes) pour masquer l'adresse IP des émetteurs et destinataires utilisant le réseau. Chaque couche d'une transaction Monero, l'émetteur, le destinataire, le montant de la transaction et (avec Kovri) la charpente de la transaction elle-même seront tous privés et confidentiels.
+
+**Portefeuille matériel :** Le portefeuille matériel Trezor Model T a ajouté Monero à sa liste de **[monnaies supportées](https://trezor.io/coins/#XMR)**. **[Le portefeuille matériel Ledger](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** est déjà disponible à travers le portefeuille GUI officiel. La communauté Monero finance une équipe qui **[construit Kastelo, un portefeuille matériel open-source](http://kastelo.org/)** développé sur l'ethos de Monero. Ces deux options de portefeuilles matériels devraient être opérationnelles d'ici la fin 2018.
+
+**Kasisto :** Cette **[solution de point de vente](https://github.com/amiuhle/kasisto)** est conçue pour faciliter les ventes pour les commerçants et les professionnels.
+
+
+## RESSOURCES ADDITIONNELLES
+
+**_[getmonero.org (Site officiel)](https://getmonero.org/)_**
+**_[monero.how](https://www.monero.how/)_**
+**_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
+**_[Guide pour Monero (post reddit)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
+**_[Introduction technique de Monero](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
+**_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**
+
+**_[Fraudes à éviter](https://www.reddit.com/r/Monero/wiki/avoid)_**
+**_[FAQ Monero](https://ww.getmonero.org/get-started/faq/)_**
```

Comment:
This also is not incorrect but we could forward them to the French FAQ link:
https://www.getmonero.org/fr/get-started/faq/